CHANGELOG_PERSISTENCE.md

```diff
@@ -0,0 +1,9 @@
+# Unreleased
+
+## Changes
+- 
+
+# persistence-v1.0.0 - 2022-09-02
```

Comment:
Did you release v1.0.0?
If not just add completed tasks under ```Unreleased Changes```
And add under version whenever release new version.

Reply:
done